tests/target/expr.rs

```diff
@@ -183,3 +183,9 @@ fn arrays() {
 
     [1 + 3, 4, 5, 6, 7, 7, fncall::<Vec<_>>(3 - 1)]
 }
+
+fn struct_exprs() {
+    Foo { a: 1, b: f(2) };
+    Foo { a: 1, b: f(2), ..g(3) };
+    LoooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooongStruct { ..base };
```

Comment:
I think we should add a line break here. Could you add a FIXME with a reference to https://github.com/nrc/rustfmt/issues/231?